no-literal-securerandom-exact-cmd.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+## This script runs wpi-many.sh with appropriate arguments and
+## environment variables. It is copied from the experiments for the
```

Comment:
I suggest deleting the sentence "It is copied...".  That information isn't useful to readers, who just want to use this script.  (And they don't know what the "Continuous Compliance paper" is, anyway.)  If you wanted to retain this script without change from the Continuous Compliance experiments, you could just provide a URL rather than copying the script.  But I think it's useful to have it here, though with changes to make it easier to use.

Then also change "re-use" to "use", and rename the script.

Reply:
This script has already been sufficiently changed that changing it further isn't a problem.